scripts/install_third_party.py

```diff
@@ -27,6 +27,11 @@
 
 import common
 
+#set the proxy to empty sting if the installation is through Vagrant so that
```

Comment:
Don't need to repeat what the code does in the comment -- and I'd prefer to avoid external links where possible. Instead, maybe something like:

```
# These two lines prevent a "IOError: [Errno socket error] [Errno -2] Name or service not known" error
# in urllib.urlretrieve, if the user is behind a proxy.
```